src/Chapter1.hs

```diff
@@ -479,7 +479,7 @@ Implement a function that returns the last digit of a given number.
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
 lastDigit n = error "lastDigit: Not implemented!"
-
+lastDigit n = (abs n) `mod` 10
```

Comment:
Nice work on using infix form of calling functions! We don't explain this syntax in our course to make it short. But well done :+1: 

Also, you can remove `()`, they are redundant in this case :slightly_smiling_face: 
```suggestion
lastDigit n = abs n `mod` 10
```